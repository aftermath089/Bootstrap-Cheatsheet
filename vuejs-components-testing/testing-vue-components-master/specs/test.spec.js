import {mount} from '@vue/test-utils'
import TestComponent from '@/test.vue'
import List from '@/list.vue'

test('mount vue component', () => {
    const wrapper = mount(TestComponent, {
        propsData: {
            value: 'VueSchool'
        }
    })
    expect(wrapper.html()).toMatchSnapshot()
})

test("ListComponent", () => {
    const wrapper = mount(List);
    const movies = wrapper.vm.marvelMovies;
    
    wrapper.setData({
        marvelMovies: [...movies, "Endgame"]
    });
    
    wrapper.vm.$nextTick( () => {
        expect(wrapper).toMatchSnapshot();
        done();
    })
})