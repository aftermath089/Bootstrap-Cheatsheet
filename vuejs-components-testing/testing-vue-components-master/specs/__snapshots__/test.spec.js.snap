// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mount vue component 1`] = `
<div>
  <h1>Hello VueSchool</h1>
</div>
`;
